src/dmd/backend/dcgcv.d

```diff
@@ -243,7 +243,7 @@ debtyp_t * debtyp_alloc(uint length)
 
 debug
 {
-    uint len = debtyp_t.sizeof - (d.data).sizeof + length;
+    const len = debtyp_t.sizeof - (d.data).sizeof + length;
```

Comment:
Maybe lose the parenthesis around `d.data` ?

Reply:
That pattern is used elsewhere, too. Better leave it for one of Walter's cleanup passes...